runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -213,6 +216,20 @@ public void onChatMessage(ChatMessage event)
 				}
 			}
 		}
+
+		if (event.getMessage().contains("Chambers of Xeric"))
```

Comment:
I think this double check and replacing can be safely replaced with just String#contains(Your completed Chambers of Xeric count is:) and then when matching with NUMBER_PATTERN do the replaceAll, probably also with regex instead of matching the color directly (as they can change the color in future for whatever reason).